Jenkinsfile.deploy

```diff
@@ -263,6 +277,13 @@ void commitAndCreatePR(String repo) {
     }
 }
 
+void commitAndCreatePRIgnoringPackageLock(String repo) {
+    dir(repo) {
+        sh 'git checkout -- */package-lock.json'
```

Comment:
Changes in `package-lock.json` should not be ignored.

Reply:
Good point. We should commit the changes as these are exact versions we built with, right?